makefiles/tools/serial.inc.mk

```diff
@@ -5,13 +5,18 @@ ifeq ($(OS),Linux)
 else ifeq ($(OS),Darwin)
   PORT ?= $(PORT_DARWIN)
 endif
-
 ifeq ($(PORT),)
   $(info Warning: no PORT set!)
 endif
+export PORT
 
 export BAUD ?= 115200
-export TERMFLAGS ?= -p "$(PORT)" -b "$(BAUD)"
-export TERMPROG ?= $(RIOTBASE)/dist/tools/pyterm/pyterm
 
-export PORT
+TERMINAL ?= pyterm
+ifeq ($(TERMINAL),pyterm)
+    export TERMPROG ?= $(RIOTBASE)/dist/tools/pyterm/pyterm
+    export TERMFLAGS ?= -p "$(PORT)" -b "$(BAUD)"
+else ifeq ($(TERMINAL),picocom)
+    export TERMPROG = picocom
```

Comment:
should we make them ```?=```, too?

Reply:
I think for `TERMPROG` its not really needed, as in the case one chooses `TERMINAL=picocom`, I guess one wants to use picocom, right? But then again, it does not hurt here. For the `TERMFLAGS` it makes sense, so will fix.